bindings/rust-examples/hyper-server-client/src/bin/client.rs

```diff
@@ -0,0 +1,67 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+use bytes::Bytes;
+use clap::Parser;
+use http_body_util::{BodyExt, Full};
+use hyper_util::{client::legacy::Client, rt::TokioExecutor};
+use std::error::Error;
+use std::str::FromStr;
+
+/// NOTE: this CA is to be used for demonstration purposes only!
+const CA: &[u8] = include_bytes!(concat!(env!("CARGO_MANIFEST_DIR"), "/../certs/ca-cert.pem"));
+
+#[derive(Parser)]
+struct Args {
+    #[clap(short, long, default_value = "localhost:1142")]
```

Comment:
Nit, mostly curious: Why 1142? I'm not sure that's a random choice I've seen before

Reply:
It's the last part of my phone number :D

Reply:
I changed it: 4bf818e00d37ce55230ba77cb05e390528525794